src/components/accountWizard/accountWizardStepOne.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import CopyField from '../copyField/copyField';
+
+const AccountWizardStepOne = ({ policySetup }) => (
+  <ul>
+    <li>
```

Comment:
Requested change, step one now has an additional name input. Validation on duplicate naming may be required